src/components/boardmember/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './style.less';
+
+interface BoardMemberProps {
+  name: string;
+  title: string;
+  photo: string;
+}
+
+const BoardMember: React.FC<BoardMemberProps> = ({ name, title, photo, }) => (
+  <div className="board-member-card">
+      <div className="board-member-photo" style={{backgroundImage: 'url('+photo+')'}} />
+      <div className="board-member-info">
+          <div className="board-member-title">
```

Comment:
Why wrap it in a div versus an h2/h3?

Reply:
tbh I thought of `h2`/`h3` as only section headers and not really headers for `div`s, it's probably better to use those elements